api/client/service/opts.go

```diff
@@ -462,7 +463,6 @@ func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 			Delay:         opts.update.delay,
 			FailureAction: opts.update.onFailure,
 		},
-		Networks:     convertNetworks(opts.networks),
```

Comment:
I understand that it is swarmkit related changes. Since this is a Spec change, I just want to make sure if this spec change will impact a mixed 1.12.1 and later versions of 1.12.x in manager and workers ?